src/common/components/card/Card.tsx

```diff
@@ -9,30 +11,37 @@ import Icon from '../icon/Icon';
 interface CardProps {
   action: () => void;
   actionText: string;
+  alt?: string;
+  className?: string;
   focalContent?: ReactNode;
   image: string;
   primaryAction?: () => void;
   primaryActionText?: string;
+  qr?: string;
   title: string;
 }
 
 const Card: FunctionComponent<CardProps> = ({
   action,
   actionText,
+  alt = '',
```

Comment:
It should be fine to have an empty alt text: If the image has no information / is only decorative it is better to send a blank alt text to screen readers.